source/_integrations/prometheus.markdown

```diff
@@ -204,3 +204,9 @@ For example:
 - record: "known_temperature_c"
   expr: "temperature_c unless entity_available == 0"
 ```
+
+## Supported metrics
+
+Metrics are exported only for the following domains:
+
+`automation`, `binary_sensor`, `climate`, `cover`, `counter`, `device_tracker`, `fan`, `humidifier`, `input_boolean`, `input_number`, `light`, `lock`, `number`, `person`, `sensor`, `update`, `switch`, `zwave`
```

Comment:
`zwave` is not an entity component or entity component platform? How can it be in this list?

Reply:
I don't have any zwave devices so I'm not sure how they work.  I just made a list of all `_handle_` functions based on the code here:
https://github.com/home-assistant/core/blob/e2f9ba5455712c3f9ecc8c4b0a5356b06bf2ce01/homeassistant/components/prometheus/__init__.py#L205

There is a function for zwave:

https://github.com/home-assistant/core/blob/e2f9ba5455712c3f9ecc8c4b0a5356b06bf2ce01/homeassistant/components/prometheus/__init__.py#L687

Reply:
To me, that sounds like a blast from the past that is a leftover from the old open Z-Wave that Home Assistant used to use many moons ago.

Reply:
Ok, then I'll just remove it from this list.  I suppose it doesn't hurt to leave it in the code for backwards compatibility.

Reply:
> Ok, then I'll just remove it from this list. 

👍 

>  I suppose it doesn't hurt to leave it in the code for backwards compatibility.

It doesn't hurt, but the code doesn't provide any backward compatibility, as all traces of the old Zwave integration have been removed from the codebase already. So, to me... it is like dead code 😄